### PR TITLE
Add new message fields.

### PR DESCRIPTION
**Pull request by mreid-moz**
Add message fields for RemoteAddr, URL Path, EnvVersion,
Hostname (the name of the server that handled the request), and
Host (the hostname to which the client sent the request).

### CODE REVIEW

File: plugins/http/http_listen_input.go [hunk 1]
```diff
@@ -71,43 +72,56 @@ func defaultStarter(hli *HttpListenInput) (err error) {
...
 func (hli *HttpListenInput) makePackDecorator(req *http.Request) func(*PipelinePack) {
 	packDecorator := func(pack *PipelinePack) {
 		pack.Message.SetType("heka.httpdata.request")
 		pack.Message.SetPid(hli.hekaPid)
 		pack.Message.SetSeverity(int32(6))
-		if field, err := message.NewField("Protocol", req.Proto, ""); err == nil {
+
+		// Host on which heka is running.
+		pack.Message.SetHostname(hli.hostname)
+		pack.Message.SetEnvVersion("1")
```
Comment (mreid-moz) [inline plugins/http/http_listen_input.go]: I'm not sure if it makes sense to arbitrarily set this to "1", but it seemed preferable to leaving it blank.

Comment (trink) [inline plugins/http/http_listen_input.go]: Blank takes up no space on the wire but I have no problem with bumping the version when we change the schema.

**Comment by trink** [comment]
Please update the CHANGES.txt

**Comment by mreid-moz** [comment]
CHANGES.txt updated.

**Comment by mreid-moz** [comment]
I updated the `RemoteAddr` and `Host` fields to remove the port info, since it doesn't seem useful, and could be cumbersome for users of these fields to have to parse out the IP.

**Pull request merged**